File: Python Word Examples/SectionsApiDemo/getSectionChildObjects.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.SectionsApi(configuration)

file_name = 'PageSetup.docx'
section_index = 0
result =api.get_section_child_objects(file_name, section_index, folder="input")

File: JavaScript Word Examples/SectionsApiDemo/getSectionChildObjects.js
(function (){
    var appId = "your id";
    var appKey = "your key";  
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.SectionsApi(apiClient);

    var fileName = 'PageSetup.docx';
    var sectionIndex = 0;
    var opt = { folder: 'input' };
    instance.getSectionChildObjects(fileName, sectionIndex, opt,
        function (error, data, response) {
            if (error) throw error;
        });
})();